Tasks/ShellScript/task.json

```diff
@@ -53,6 +53,15 @@
             "groupName": "advanced"
         },
         {
+            "name": "disableAutoCwd",
+            "type": "boolean",
+            "label": "Disable Auto CWD",
+            "defaultValue": "false",
```

Comment:
...if you tie to the visibility, you might also want to put the disableAutoCwd checkbox above the cwd input.

I believe this rule on the cwd input would suffice:
```
"visibleRule": "disableAutoCwd = true",
```